ui/packages/components/src/DatePicker/Calendar.tsx

```diff
@@ -24,7 +29,7 @@ export function Calendar({ selected, onSelect }: CalendarProps) {
 
 const classNames: DayPickerDefaultProps['classNames'] = {
   caption: 'flex justify-center items-center h-6',
-  root: 'text-slate-900',
+  root: 'text-slate-900 dark:bg-white',
```

Comment:
Did you mean `bg-white` ? I think the dark mode of this will be darker

Reply:
Yeah, I was hard coding bg-white because the calendar picker would otherwise be illegible in dark mode. I'll switch to just `bg-white`.

Reply:
done